index.html

```diff
@@ -95,5 +95,179 @@ <h2>
         </button>
     </div>
   </section>
```

Comment:
- [ ] [REQUIRED] As the previous code Reviewer suggested. All of your project sections should be within a HTML main tag and give it the main container a `display: grid`

EXAMPLE

`<main class="main-container">`
    `<section>`
     ...
    `</section>`


`</main>`

.main-container {
 display: grid;
}

Reply:
Thank you for your kind response. display: grid; is already added in CSS file under work container.

Reply:
![asdf](https://user-images.githubusercontent.com/77225127/159097099-f183bd69-5408-41ce-b0b7-c1a80d8e9568.PNG)
You can see that the first reviewer admitted that I already added  display: grid;

Reply:
Please Kindly surround the main sections of the page with a main tag, then submit for a rereview. This is better for [semantics](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main). The  main container should also have a display grid